packages/bootstrap/scss/treelist.scss

```diff
@@ -1,12 +1,12 @@
-@import "variables";
-@import "mixins";
+@import "_variables.scss";
+@import "_mixins.scss";
 
 
 // Dependencies
-@import "common/all";
-@import "grid";
+@import "common/_all.scss";
+@import "grid.scss";
 
 
 // Component
-@import "treelist/layout";
-// @import "treelist/theme";
+@import "treelist/_layout.scss";
+// @import "treelist/_theme.scss";
```

Comment:
Do we need this?

Reply:
That needs revision for sure. Still the purpose of the PR is to switch to fully qualified names